### ISSUE TITLE
letsencrypt-renewer not found

### ISSUE THREAD

**Issue by jnovermars** [alexalouit/ISPConfig-letsencrypt#38]
He i am not sure if its realy a bug, but when i install Letsencrypt on the way its documented it give a error that the Letsencrypt-renew is not installed:

Installed following the instructions of:
https://github.com/letsencrypt/letsencrypt 

Error (CLI output):
`root@server:/tmp/ISPConfig-letsencrypt# php -q install.php
PHP Deprecated:  Comments starting with '#' are deprecated in /etc/php5/cli/conf.d/ming.ini on line 1 in Unknown on line 0`

`Create backup on /var/backup/ directory
/bin/tar: Removing leading /' from member names
Backup finished`

`ERROR: Let's Encrypt ( /root/.local/share/letsencrypt/bin/letsencrypt-renewer ) is missing, install it corecctly!`

`root@server:/tmp/ISPConfig-letsencrypt# letsencrypt renew
bash: letsencrypt: command not found`

i hope you have a solution for this

**Comment by joey91133** [comment]
They change renew to command.
https://github.com/letsencrypt/letsencrypt/issues/2376

**Comment by cvladan** [comment]
```
sed -i -re 's/letsencrypt-renewer/letsencrypt renew/gi' install.php
php -q install.php
```

should work :)

**Comment by lucatze** [comment]
Hm. I tried your command, but it didn't work. I checked the file, its definitely present at the specified path. The sed command apparently worked also.

```
root@server:~/ISPConfig-letsencrypt# php -q install.php
Create backup on /var/backup/ directory
/bin/tar: Removing leading `/' from member names
Backup finished
ERROR: Let's Encrypt ( /root/.local/share/letsencrypt/bin/letsencrypt renew ) is missing, install it correctly!
root@server:~/ISPConfig-letsencrypt#
```

```
root@server:~# .local/share/letsencrypt/bin/letsencrypt renew

No renewals were attempted.
root@server:~#
```

The install.php Lines:

```
if(!is_file("/root/.local/share/letsencrypt/bin/letsencrypt renew")) {
        echo "ERROR: Let's Encrypt ( /root/.local/share/letsencrypt/bin/letsencrypt renew ) is missing, install it co$        exit;
}
```

**Comment by andypl78** [comment]
Try
sed -i -re 's/letsencrypt-renewer/letsencrypt/gi' install.php

**Comment by genogod** [comment]
> Try
> sed -i -re 's/letsencrypt-renewer/letsencrypt/gi' install.php

This worked for me, thanks!!

**Comment by alexalouit** [comment]
fix in 2a9b20e
